print/print-apps/geoportailv3/config.yaml

```diff
@@ -1,49 +1,349 @@
 pdfConfig: !pdfConfig
   compressed: false
   author: "Mapfish Print"
-  subject: "Mapfish Print"
+  subject: "Mapfish Print Verbose example"
   creator: "Mapfish Print"
-  keywords: ["legend", "table", "tablelist", "map"]
 
 throwErrorOnExtraParameters: true
 
 templates:
   A4 portrait: !template
     pdfConfig: !pdfConfig
-      title: "A4 Portrait"
-    reportTemplate: report.jrxml
-    tableData: jrDataSource
+      title: "a4_portrait"
```

Comment:
OK, I think I understood. What will be translated is the name of the template. This one is going into the PDF.
I'll update them